src/components/AppNavigation/AppNavigation.vue

```diff
@@ -45,7 +45,7 @@
         <icon-access-control />Access Control
         <icon-expand class="icon-expand" />
       </b-button>
-      <b-collapse tag="ul" id="access-control-menu" class="nav-item__nav">
+      <b-collapse id="access-control-menu" tag="ul" class="nav-item__nav">
```

Comment:
Will we get build errors when attributes aren't in the right order?

Reply:
Yes, but running `npm run lint` auto fixes them. I expect they will be auto fixed if you have your editor set to auto fix eslint on save.